### DISCUSSION TITLE
关于简悦 UI&UX 的相关讨论

### DISCUSSION THREAD

**Discussion by Joel2561** [Kenshin/simpread#3107] [讨论组]
计划整理一些简悦界面和用户交互相关的问题，提出我个人的一些看法。
当然，这些“问题”全部都是我个人的见解，欢迎作者和大家一起来讨论。

**Comment by Joel2561** [comment]
## 插件管理页面和插件中心页面中插件卡片的文本调整
插件卡片中第一行是对用户相对帮助不大的插件作者姓名，插件名称放到了第二行。
建议将插件名称放到第一行，将插件简介放至第二行，将作者姓名放至右下角，或隐去作者，在插件的详细信息中显示作者。
![](https://user-images.githubusercontent.com/77143461/148707703-b0e14700-8e15-45f9-bf18-c25edaf5fa6f.png)

**Comment by Joel2561** [comment]
## 插件中心页面中插件卡片的触发
插件中心的插件卡片只有动效，没有链接到插件的详细说明页面。
建议：将右下角三个点触发详细说明页面改为点击整个插件卡片都能触发跳转到插件详细说明页。

**Comment by Joel2561** [comment]
插件中心插件卡片中下方第二个图标只能显示链接，并不能直接跳转或复制。
建议：点击第二个图标后可以直达链接。
![image](https://user-images.githubusercontent.com/77143461/148708191-a8912592-2e92-457c-873f-81fc8e891b6e.png)

**Comment by Joel2561** [comment]
## 选项页高级模式切换到简易模式后不易找到切换的开关
当切换到简易模式后，用户不容易找到切换回高级模式的开关，因为他藏在了左下角，而且会被左栏设定TOC全部挡住。如下图：
![image](https://user-images.githubusercontent.com/77143461/150722782-0bdf8fb9-43fa-443e-906e-e749b8200713.png)
***
![image](https://user-images.githubusercontent.com/77143461/150722860-e1afc905-58a0-4394-b7b8-c46d93d44868.png)

**Reply by Joel2561** [reply]
另外简易模式下共通选项下的配置文件栏有一个小注有别字：
![image](https://user-images.githubusercontent.com/77143461/150723111-7cad677f-708a-4968-a21b-01203da5e1c9.png)

**Reply by Kenshin** [reply]
哈哈，已经有用户反馈过了。  😂

**Comment by Joel2561** [comment]
## 阅读模式中图片显示策略
在阅读模式中，图片经过渲染后会加上一层边框，然而，有时这层边框会约束图片和行文的连贯性。

### 渲染前
![image](https://user-images.githubusercontent.com/77143461/158026207-36aa1182-9b76-401b-a7b7-eb8cbe2af9a7.png)

### 渲染后
![image](https://user-images.githubusercontent.com/77143461/158026226-b1bf5c8e-9872-469c-bc7a-f39122fe06d6.png)

## 改进
针对阅读模式中图片的显示添加选项，让用户自由选择图片是否具有边框。也可增加图片阴影的功能。

**Reply by Joel2561** [reply]
另外图中关于图片位置的语句好像被识别成文本了？

**Reply by Kenshin** [reply]
这个很难判断，简悦目前的做法是根据自己的一套算法来判断，但肯定会有误判的情况，但目前来说比较少见了。

大多发生在你举例子的场景中。

**Reply by Joel2561** [reply]
是的，但图片边框的问题……催更😊

**Comment by Kenshin** [comment]
**原因** 右下角的“帮助中心”和“给我反馈”两个按钮，展开卡片默认前者置于后者之上
**背景** 卡片需要手动点击原button关闭不能自动关闭的背景
**结果** 先点击_帮助中心_再点击_给我反馈_，会导致后者卡片被部分遮挡。

via https://github.com/Kenshin/simpread/issues/3568
